Realm.xcodeproj/project.pbxproj

```diff
@@ -2858,6 +2876,9 @@
 			isa = XCBuildConfiguration;
 			baseConfigurationReference = 5D659E761BE03E0D006515A0 /* Realm.xcconfig */;
 			buildSettings = {
+				CLANG_ENABLE_MODULES = YES;
```

Comment:
None of these changes should be here. If there are any changes to the project settings which should happen, they should be done in the xcconfig file.